### PR TITLE
prep `2.14.18`

### PR DESCRIPTION
**Pull request by zzstoatzz**
cutting an early patch release to get [this](https://github.com/PrefectHQ/prefect/pull/11758) fix out asap

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+## New Contributors
+* @clefelhocz2 made their first contribution in https://github.com/PrefectHQ/prefect/pull/11722
+### Contributors
+- @abrookins
+- @clefelhocz2
+- @zhen0
+- @zzstoatzz
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.14.17...preview
```
Comment (serinamarie) [inline RELEASE-NOTES.md:23]:
```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.14.17...2.14.18
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+### Uncategorized
+- Clarify that interval schedules use an anchor not start date — https://github.com/PrefectHQ/prefect/pull/11767
+
+## New Contributors
+* @clefelhocz2 made their first contribution in https://github.com/PrefectHQ/prefect/pull/11722
+### Contributors
+- @abrookins
+- @clefelhocz2
+- @zhen0
+- @zzstoatzz
```
Comment (serinamarie) [inline RELEASE-NOTES.md:21]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+## Release 2.14.18
+
+### Fixes
+- Fix environment variable not accepting lists — https://github.com/PrefectHQ/prefect/pull/11722
+- Revert deprecation of worker webserver setting — https://github.com/PrefectHQ/prefect/pull/11758
+
+### Documentation
+- Detail `send_input` and `receive_input` — https://github.com/PrefectHQ/prefect/pull/11724
+
+### Uncategorized
```
Comment (serinamarie) [inline RELEASE-NOTES.md:12]:
```suggestion
```

effectively moves this to docs section where it belongs

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.14.18
+
+### Fixes
+- Fix environment variable not accepting lists — https://github.com/PrefectHQ/prefect/pull/11722
+- Revert deprecation of worker webserver setting — https://github.com/PrefectHQ/prefect/pull/11758
+
+### Documentation
+- Detail `send_input` and `receive_input` — https://github.com/PrefectHQ/prefect/pull/11724
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Expand docs on interactive flows, detailing `send_input` and `receive_input` — https://github.com/PrefectHQ/prefect/pull/11724
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.14.18
+
+### Fixes
+- Fix environment variable not accepting lists — https://github.com/PrefectHQ/prefect/pull/11722
```
Comment (serinamarie) [inline RELEASE-NOTES.md:6]: This should maybe just be "allow prefect settings to accept lists?" i think?

**Review by serinamarie — APPROVED** [review]

File: RELEASE-NOTES.md (hunk 5 above)
Comment (zzstoatzz) [inline RELEASE-NOTES.md:6]:
```suggestion
- Allow prefect settings to accept lists — https://github.com/PrefectHQ/prefect/pull/11722
```

Comment (zzstoatzz) [reply RELEASE-NOTES.md]: update in [e5fdd2b](https://github.com/PrefectHQ/prefect/pull/11769/commits/e5fdd2bc9c96062aadea25e8c81df4948227afaa)

**Pull request merged**